docs/guides/server/hostname.adoc

```diff
@@ -85,42 +85,276 @@ To reduce attack surface, the administration endpoints for Keycloak and the Admi
 
-=== Exposing the server behind a TLS termination proxy
 
-In this example, the server is running behind a TLS termination proxy and publicly available from `https://mykeycloak`.
+Through this set of hostname configuration options, Keycloak aims to provide a flexible way to adapt to the characteristics of different environments.
 
-.Configuration:
-<@kc.start parameters="--hostname=mykeycloak --proxy=edge"/>
+Misconfiguration of the Hostname and/or its Reverse Proxy options can lead to unusual and unpleasant behavior, such as Keycloak's Administration Console not loading correctly.
 
-=== Exposing the server without a proxy
+In an attempt to further improve the user experience and minimize the occurrence of misconfigurations, see the list of sample Hostname and Proxy configurations below, which is oriented to possible scenarios that may exist in different computing environments.
 
-In this example, the server is running without a proxy and exposed using a URL using HTTPS.
+The following are more example scenarios and the corresponding commands for setting up the `Hostname` and/or `Proxy` options.
 
-.Keycloak configuration:
-<@kc.start parameters="--hostname-url=https://mykeycloak"/>
+> Note that the `start` command requires setting up TLS. The corresponding options are not shown for example purposes. For more details, see <@links.server id="enabletls"/>.
+
+
+=== Scenario 01 - Beginner/Quick Test in Localhost with HTTP only
+
+* __**Description**__:
+  ** The most basic option to run, in localhost, all default values.
+* __**Recommended Settings / Keycloak configuration**__:
+
+  <@kc.startdev parameters="" />
+
+* __**Expected Behavior**__:
+  ** Both Frontend/Backend and Admin Console at: http://localhost:8080
+
+---
+
+
+=== Scenario 02 - Custom Hostname with HTTP only
+
+* __**Description**__:
+** Just setting a custom Hostname to be reachable via HTTP only.
+* __**Recommended Settings / Keycloak configuration**__:
+
+  <@kc.start parameters="--hostname=mykeycloak.com --http-enabled=true --hostname-strict-https=false" />
+
+* __**Expected Behavior**__:
+  ** Both Frontend/Backend and Admin Console at: http://mykeycloak.com:8080
+
+---
+
+
+=== Scenario 03 - Custom Hostname over HTTPS
+
+* __**Description**__:
+** Setting a custom Hostname to be reachable via HTTPS and exposing the server without a proxy. In this example, the server is running without a proxy and exposed using a URL using HTTPS.
+* __**Recommended Settings / Keycloak configuration**__:
+
+** Set the HTTPS/TLS Certificates in Keycloak. For more details, see <@links.server id="enabletls"/>.
+
+  <@kc.start parameters="-hostname=mykeycloak.com --https-certificate-file=/path-to-file/the-cert.pem --https-certificate-key-file=/path-to-file/the-cert-key.pem" />
+
+* __**Expected Behavior**__:
+** Both Frontend/Backend and Admin Console at: https://mykeycloak.com:8443
+
+---
+
+
+=== Scenario 04 - Custom Hostname + Diff. AdminConsole URL with HTTP only
```

Comment:
> Custom Hostname + Diff. AdminConsole URL with HTTP only

Again, this is something, what I would definitely rephrase, "+ Diff. AdninConsole" doesn't seem right to me and I think we don't use similar structure anywhere else. I would stick with the format that we already use all over the guides.

Reply:
Improvement done. Please review again If it is better this way.